cmd/tailscale/cli/web.go

```diff
@@ -134,6 +136,10 @@ func runWeb(ctx context.Context, args []string) error {
 		return nil
 	}
 
+	if distro.Get() == distro.Unraid {
```

Comment:
Works for me. I was more thinking from the security side (not letting users run `tailscale web` in HTTP mode since it doesn't have an authentication path then... CGI mode allows the existing Web UI to handle authentication before invoking Tailscale). I can see your point though :)